tests/components/homewizard/test_switch.py

```diff
@@ -26,7 +26,7 @@
 ]
 
 
-@pytest.mark.parametrize("device_fixture", ["device-HWE-SKT.json"])
+@pytest.mark.parametrize("device_fixture", ["HWE-SKT"])
```

Comment:
Not sure if this adds anything useful, but HWE-P1, SDM230/SDM630 also support `cloud_enabled`.

Reply:
I tried to to broaden the scope of this PR. As in, the fixtures can be extended, and the tests as well. But the changeset is already quite large :)